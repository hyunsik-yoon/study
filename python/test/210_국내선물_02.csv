거래일자,거래번호,거래종류,적요명,종목번호,종목명,수량,단가,거래금액,수수료,예수금전잔,예수금금잔,정산금액,약정금액,세금합계,대용잔고,기타수표금액,미수/변제,대체금융회사,계좌명,받는분 표시내용,보내는분 표시내용
2023-03-30,2,매수,당일차금매수출금                                                                ,1foo4000,foobar   F 202304 (  10)                                                                                                                                                                                                                                                                                  ,120,"218,000","6,600,000","25,300","272,184,300","265,559,000","6,625,300","261,600,000",0,0,0,0,                                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
2023-03-31,1,입금,매수갱신차금입금                                                                ,1foo4000,foobar   F 202304 (  10)                                                                                                                                                                                                                                                                                  ,120,0,"7,200,000",0,"265,559,000","272,759,000","7,200,000",0,0,0,0,0,                                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
2023-04-03,1,입금,매수갱신차금입금                                                                ,1foo4000,foobar   F 202304 (  10)                                                                                                                                                                                                                                                                                  ,120,0,"5,400,000",0,"277,259,000","282,659,000","5,400,000",0,0,0,0,0,                                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
2023-04-04,2,출금,매수갱신차금출금                                                                ,1foo4000,foobar   F 202304 (  10)                                                                                                                                                                                                                                                                                  ,120,0,"4,800,000",0,"239,159,000","234,359,000","4,800,000",0,0,0,0,0,                                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
2023-04-05,1,입금,매수갱신차금입금                                                                ,1foo4000,foobar   F 202304 (  10)                                                                                                                                                                                                                                                                                  ,120,0,"19,200,000",0,"234,359,000","253,559,000","19,200,000",0,0,0,0,0,                                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
2023-04-06,1,입금,매수갱신차금입금                                                                ,1foo4000,foobar   F 202304 (  10)                                                                                                                                                                                                                                                                                  ,120,0,"6,600,000",0,"232,559,000","239,159,000","6,600,000",0,0,0,0,0,                                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
2023-04-07,2,출금,매수갱신차금출금                                                                ,1foo4000,foobar   F 202304 (  10)                                                                                                                                                                                                                                                                                  ,120,0,"7,200,000",0,"210,659,000","203,459,000","7,200,000",0,0,0,0,0,                                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
2023-04-07,3,매도,당일차금매도출금                                                                ,1foo4000,foobar   F 202304 (  10)                                                                                                                                                                                                                                                                                  ,30,"234,000","150,000","6,780","203,459,000","203,302,220","156,780","70,200,000",0,0,0,0,                                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
2023-04-10,1,입금,매수갱신차금입금                                                                ,1foo4000,foobar   F 202304 (  10)                                                                                                                                                                                                                                                                                  ,90,0,"7,200,000",0,"203,302,220","210,502,220","7,200,000",0,0,0,0,0,                                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
2023-04-11,1,입금,매수갱신차금입금                                                                ,1foo4000,foobar   F 202304 (  10)                                                                                                                                                                                                                                                                                  ,90,0,"11,250,000",0,"225,535,533","236,785,533","11,250,000",0,0,0,0,0,                                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    
2023-04-11,3,매도,당일차금매도출금                                                                ,1foo4000,foobar   F 202304 (  10)                                                                                                                                                                                                                                                                                  ,90,"244,500","9,450,000","21,280","238,285,533","228,814,253","9,471,280","220,050,000",0,0,0,0,                                                                                ,                                                                                                                                                                                                ,                                                                                                    ,                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    

