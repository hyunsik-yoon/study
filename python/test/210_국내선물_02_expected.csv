종목명,진입일자,진입단가,청산일자,청산단가,거래수량,총수수료,총세금합계,순수익
foobar   F 202304 (  10),03/30/2023,218000,04/07/2023,234000,30,13105,0,4800000
foobar   F 202304 (  10),03/30/2023,218000,04/11/2023,244500,90,40255,0,23850000
